openshift/datasync-showcase.yml

```diff
@@ -5,10 +5,10 @@ kind: Template
 labels:
   template: ionic-showcase-server
```

Comment:
In openshift, the name in the UI is appearing as "ionic-showcase-server".
I'm think that it would make more sense to set this value to "Ionic Showcase Server".